common/ui.lua

```diff
@@ -10,14 +11,33 @@ function InitUI()
         imgCheckbox = imgButtonShortBlank
     end
 
+    TextColor = {
+        --blackHalfOpacity = 0x0000007f
+        LightBrown = { R = 188, G = 168, B = 140 },
+        LightGreen = { R = 24, G = 255, B = 0 },
+        Black = { R = 255, G = 255, B = 255 },
```

Comment:
This is white, not black. Black on buttons come from the "multiply" blend mode

Reply:
uugh

Reply:
fixed